src/lib/stepper/stepper-vertical.html

```diff
@@ -1,24 +1,19 @@
 <div class="mat-step" *ngFor="let step of _steps; let i = index; let isLast = last">
-  <div #stepHeader class="mat-vertical-stepper-header" role="tab"
-       [id]="_getStepLabelId(i)"
-       [attr.aria-controls]="_getStepContentId(i)"
-       [attr.aria-selected]="selectedIndex == i"
-       [tabIndex]="_focusIndex == i ? 0 : -1"
-       (click)="step.select()"
-       (keydown)="_onKeydown($event)">
-    <div class="mat-stepper-index">
-      {{i + 1}}
-    </div>
-
-    <div class="mat-stepper-label">
-      <!-- If there is a label template, use it. -->
-      <ng-container *ngIf="step.stepLabel"[ngTemplateOutlet]="step.stepLabel.template">
-      </ng-container>
-      <!-- It there is no label template, fall back to the text label. -->
-      <div *ngIf="!step.stepLabel">{{step.label}}</div>
-    </div>
-
+  <div class="mat-stepper-header" #stepperHeader (click)="step.select()"
```

Comment:
you can remove this div and get access to the `md-step-header` element like this:

```ts
@ViewChildren(MdStepHeader, {read: ElementRef}) _stepHeaders;
```

(and same for horizontal)